__tests__/libs/ontuml2gufo/annotations.test.ts

```diff
@@ -25,16 +25,45 @@ describe('Relations', () => {
   let result;
 
   beforeAll(async () => {
-    result = await transformOntoUML2GUFO(istandard);
+    result = await transformOntoUML2GUFO(annotations);
   });
 
-  it('should generate "nl" label', async () => {
-    expect(result).toContain('<:LegalGender> <rdfs:label> "Geslacht"@nl');
+  it('should generate language labels on classess (testing with nl, pt, en, and it)', async () => {
+    expect(result).toContain('<:Person> <rdfs:label> "Persoon"@nl');
+    expect(result).toContain('<:Person> <rdfs:label> "Pessoa"@pt');
+    expect(result).toContain('<:Person> <rdfs:label> "Person"@en');
+    expect(result).toContain('<:Person> <rdfs:label> "Persona"@it');
   });
 
-  it('should generate "description" as a rdfs:comment', async () => {
+  // it('should generate language labels on attributes (testing with nl, pt, en, and it)', async () => {
```

Comment:
Please fix the transformation so that these tests will pass.
Attributes must also be considered when generating labels and comments.

Reply:
Done